templates/dockerdiscordcontrol.xml

```diff
@@ -11,35 +11,59 @@
   <Icon>https://raw.githubusercontent.com/DockerDiscordControl/ddc-website/main/DDClogo.jpg</Icon>
-  <Screenshot>https://raw.githubusercontent.com/DockerDiscordControl/ddc-website/main/webui1.png</Screenshot>
-  <Screenshot>https://raw.githubusercontent.com/DockerDiscordControl/ddc-website/main/commands.png</Screenshot>
-  <Screenshot>https://raw.githubusercontent.com/DockerDiscordControl/ddc-website/main/control1.png</Screenshot>
-  <Screenshot>https://raw.githubusercontent.com/DockerDiscordControl/ddc-website/main/task1.png</Screenshot>
-  <DonateText>Support DDC Development - Buy Me A Coffee or PayPal</DonateText>
+  <Screenshot>https://raw.githubusercontent.com/DockerDiscordControl/ddc-website/main/images/D_ServerOverview.png</Screenshot>
+  <Screenshot>https://raw.githubusercontent.com/DockerDiscordControl/ddc-website/main/images/D_AdminOverview.png</Screenshot>
+  <Screenshot>https://raw.githubusercontent.com/DockerDiscordControl/ddc-website/main/images/D_ControlContainer.png</Screenshot>
+  <Screenshot>https://raw.githubusercontent.com/DockerDiscordControl/ddc-website/main/images/D_CreateTask.png</Screenshot>
+  <Screenshot>https://raw.githubusercontent.com/DockerDiscordControl/ddc-website/main/images/D_LiveLog.png</Screenshot>
+  <Screenshot>https://raw.githubusercontent.com/DockerDiscordControl/ddc-website/main/images/WU_ContainerSettings.png</Screenshot>
+  <DonateText>Support DDC Development - Buy Me A Coffee</DonateText>
   <DonateLink>https://buymeacoffee.com/dockerdiscordcontrol</DonateLink>
-  <Config Name="Web UI Port" Target="8374" Default="8374" Mode="tcp" Description="Port for DDC web interface." Type="Port" Display="always" Required="true" Mask="false">8374</Config>
-  <Config Name="Flask Secret Key" Target="FLASK_SECRET_KEY" Default="" Mode="" Description="Secret key for web UI security. Generate with: openssl rand -hex 32" Type="Variable" Display="always" Required="true" Mask="true"/>
-  <Config Name="Admin Password" Target="DDC_ADMIN_PASSWORD" Default="admin" Mode="" Description="Password for web UI admin user (username: admin). CHANGE FROM DEFAULT!" Type="Variable" Display="always" Required="true" Mask="true">admin</Config>
+  <Config Name="Web UI Port" Target="9374" Default="8374" Mode="tcp" Description="Port for DDC web interface (host port, mapped to container port 9374)." Type="Port" Display="always" Required="true" Mask="false">8374</Config>
+  <Config Name="Flask Secret Key" Target="FLASK_SECRET_KEY" Default="" Mode="" Description="Secret key for web UI security. RECOMMENDED: Generate with 'openssl rand -hex 32' for persistent sessions. If not set, a random key is generated (sessions reset on restart)." Type="Variable" Display="always" Required="false" Mask="true"/>
+  <Config Name="Admin Password" Target="DDC_ADMIN_PASSWORD" Default="setup" Mode="" Description="Password for web UI admin user (username: admin). CHANGE FROM DEFAULT!" Type="Variable" Display="always" Required="true" Mask="true">setup</Config>
```

Comment:
Why provide a default at all? Why not force people to fill out this field on their own?